Sources/Networking/Operations/GetIntroEligibilityOperation.swift

```diff
@@ -86,40 +74,28 @@ private extension GetIntroEligibilityOperation {
                                                      fetchToken: self.receiptData.asFetchToken)),
                                   path: .getIntroEligibility(appUserID: appUserID))
 
-        httpClient.perform(request, authHeaders: self.authHeaders) { (response: HTTPResponse<[String: Any]>.Result) in
-            let eligibilityResponse = IntroEligibilityResponse(result: response.mapError { $0.asPurchasesError },
-                                                               productIdentifiers: self.productIdentifiers,
-                                                               unknownEligibilityClosure: unknownEligibilityClosure,
-                                                               completion: self.responseHandler)
-            self.handleIntroEligibility(response: eligibilityResponse)
+        httpClient.perform(
+            request, authHeaders: self.authHeaders
+        ) { (response: HTTPResponse<GetIntroEligibilityResponse>.Result) in
```

Comment:
Replaced `HTTPResponse<[String: Any]>` with `HTTPResponse<GetIntroEligibilityResponse>` to get automatic decoding 🎉